recipes/UrbanSound8k/README.md

```diff
@@ -54,7 +54,41 @@ python train.py hparams/train_ecapa_tdnn.yaml --data_folder=/localscratch/UrbanS
 python train.py hparams/train_ecapa_tdnn.yaml --data_folder=/localscratch/UrbanSound8K/ --train_fold_nums=[1, 2, 3, 4, 5, 6, 7, 8, 9] --valid_fold_nums=[10] --test_fold_nums=[10] --output_folder=./results/urban_sound/fold_10
 `
 
-Note that the results for 10-fold must be compiled from the output folders and averaged manually.
+Note that the results for 10-fold must be compiled from the output folders and averaged using ./SoundClassification/harry_plotter.py (which computes average and displays confusions matrix using seaborn) or manually.
+
+# Perfomance without data augmentation (10 folds)
+
+* Results averaged using [harry_plotter.py](https://github.com/ZiRaymondal/speechbrain/tree/develop/recipes/UrbanSound8k/SoundClassification/harry_plotter.py)
```

Comment:
This should be a relative link within the same repo, not a link to your external repository.